C# Examples/EposServiceConsumer/UnitTests/OldEposTest.cs

```diff
@@ -0,0 +1,376 @@
+            const string applicationVersion = "1.0.0.0";
+            
+            _client = new EposServiceClient(serviceBase, consumerKey, consumerSecret, secondSecret, oAuthEndpoint, applicationName, applicationVersion);
+        }
+
+        [Test]
+        public void GetRestaurant()
+        {
+            Restaurant restaurant = _client.GetRestaurant(_targetRestaurantId);
+            Assert.IsNotNull(restaurant);
+        }
+
+        [Test]
+        [Ignore("This test depends on there being bookings in the diary. This may not always be the case.")]
+        public void GetDiaryData()
+        {
+            DiaryBooking[] diaryBookings = _client.GetDiaryData(_targetRestaurantId, DateTime.Today);
+            Assert.IsNotNull(diaryBookings);
+        }
+
+        [Test]
+        [Ignore("This test depends on there being bookings in the diary. This may not always be the case.")]
+        public void GetBooking()
+        {
+            DiaryBooking[] diaryBookings = _client.GetDiaryData(_targetRestaurantId, DateTime.Now.Date.AddDays(0));
+            Booking booking = _client.GetBooking(_targetRestaurantId, diaryBookings[0].BookingId);
+            Assert.IsNotNull(booking);
+        }
+
+        [Test]
+        [Ignore("This test depends on there being bookings in the diary. This may not always be the case.")]
+        public void AddBooking()
+        {
+            DateTime targetDateTime = DateTime.Now.Date.AddHours(12).AddMinutes(30);
+            
+            //Restaurant restaurant = _client.GetRestaurant(_targetRestaurantId);
+
+            //Segment currentSegment = Utilities.FindSegmentByDate(restaurant.Segments,targetDateTime);
+
+            //if(currentSegment == null)
+            //    throw  new Exception("Segment not found");
+
+            //Table table = currentSegment.Tables.FirstOrDefault();
+
+            //if (table == null)
+            // throw  new Exception("Table cannot be null");
+
+            var testBooking = new Booking
+                                  {
+                                      AreaId = 3780,
+                                      ArrivalStatus = 0,
+                                      BookingId = 0,
+                                      BookingReference = null,
+                                      ConfirmedByPhone = false,
+                                      Customer = new Customer { FirstName = "Tam", Surname = "Bam", AccessCode = "12345"},
+                                      Comments = null,
+                                      Covers = 2,
+                                      Duration = 120,
+                                      IsLeaveTimeConfirmed = false,
+                                      MealStatus = 4,
+                                      MenuId = 0,
+                                      ServiceId = 5534,
+                                      RestaurantId = _targetRestaurantId,
+                                      VisitDateTime = targetDateTime,
```

Comment:
Can you double check if this removes the timezone offset from the visit datetime? If not we'll be giving people an example of what not to do!

Maybe it's possible to change the `EposServiceClient` class to make sure the timezone offset is removed?

I think the way we normally do it is by using `DateTime.SpecifyKind()` and marking the date as UTC. I think this is an example: https://github.com/ResDiary/ResDiary-OAuth-Examples/pull/3/files#diff-fc01cefa5525025ea8e75162a8da8909R766